### DISCUSSION TITLE
Security: Disable JIT

### DISCUSSION THREAD

**Discussion by Rudxain** [yokoffing/Betterfox#299] [Ideas]
`Securefox` should disable JIT-compilation, including [Ion](https://wiki.mozilla.org/IonMonkey).

To avoid clashing with `Fastfox`'s speed in the main `user.js`, it should be opt-in

**Comment by yokoffing** [comment]
I will add these prefs as an option in the next release.

Mozilla should add a [Security Mode (like in Edge)](https://support.microsoft.com/en-us/microsoft-edge/enhance-your-security-on-the-web-with-microsoft-edge-b8199f13-b21b-4a08-a806-daed31a1929d) that auto-activates in Private Browsing and heuristically in Normal Browsing. One can wish!

**Comment by yokoffing** [comment]
https://github.com/yokoffing/Betterfox/blob/c5fca2dbf7289c8dbce901c040683f3cdfdd7926/Securefox.js#L1131-L1162
